packages/eslint-plugin-testing-library-react/docs/rules/prefer-expect-query-by.md

```diff
@@ -0,0 +1,28 @@
+# Avoid using `expect(getBy*)` (prefer-expect-query-by)
+
+React (DOM) Testing Library support two types of queries: `getBy*` and `queryBy*`. Using `getBy*` throws an error in case the element is not found. This is useful when using method like `waitForElement`, which are `async` functions and therefore throwing an error will cause the test to stop.
```

Comment:
That is not quite accurate. According to RTL doc (https://testing-library.com/docs/dom-testing-library/api-async#waitforelement) error thrown by `getBy*` will cause `waitForElement` to keep waiting for element to appear, until timeout happens.

Usage of `getBy*` even have some advantages in scope of `waitForElement`. With `getBy*` we can wait for several elements in a single `waitForElement`, like this:

```
const [elementA, elementB] = await waitForElement(() => [getByText('elementA'), getByText('elementB')])```
```
, which is not possible with `queryBy`.

Reply:
Yes right, I'll clarify that point. Thanks for pointing that out 👌